docs/pages/versions/unversioned/sdk/audio-av.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Audio (expo-av)
+description: A library that provides an API to implement audio playback and recording in apps.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-av'
+packageName: 'expo-av'
+iconUrl: '/static/images/packages/expo-av.png'
+platforms: ['android', 'ios', 'tvos', 'web']
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { SnackInline } from '~/ui/components/Snippet';
+import { PlatformTags } from '~/ui/components/Tag';
+
+`Audio` from `expo-av` allows you to implement audio playback and recording in your app.
```

Comment:
```suggestion
> **info** The `Audio` component from `expo-av` documented on this page will be replaced by an improved version in `expo-audio` in an upcoming release when the new library is stable. [Learn about `expo-audio`](https://docs.expo.dev/versions/unversioned/sdk/audio/).

`Audio` from `expo-av` allows you to implement audio playback and recording in your app.
```

Maybe we should add a callout here about the upcoming Audio library, just like we do for Video [(reference)](http://localhost:3002/versions/unversioned/sdk/audio/).

![CleanShot 2024-10-23 at 22 40 29](https://github.com/user-attachments/assets/4e3234a9-d1ca-404e-811c-413b08b8e626)